test/benchmarks/README.md

```diff
@@ -17,4 +23,21 @@ You can specify which components and processes to benchmark with the following f
 **--ci** will cause the the benchmarks to run on a smaller slice of each dataset and a smaller subset of Retriever / Reader / DocStores. 
 
 **--update-json** will cause the script to update the json files in docs/_src/benchmarks so that the website benchmarks will be updated.
- 
\ No newline at end of file
+ 
+**--save_markdown** save results additionally to the default csv also as a markdown file
+```
+
+Results will be stored in this directory as
+- retriever_index_results.csv (+ .md)
+- retriever_query_results.csv (+ .md)
+- reader_results.csv (+ .md)
+
+
+# Temp. Quickfix for bigger runs
+
+For bigger indexing runs (500k docs) the standard elastic / opensearch container that we spawn via haystack might run OOM. 
+Therefore, start them manually before you trigger the benchmark script and assign more memory to them: 
+
+`docker start opensearch > /dev/null 2>&1 || docker run -d -p 9201:9200 -p 9600:9600 -e "discovery.type=single-node" -e "OPENSEARCH_JAVA_OPTS=-Xms4096m -Xmx4096m" --name opensearch opensearchproject/opensearch:2.2.1`
+and
```

Comment:
Having a line break either side of this "and" will make the paragraph be much more readable